app/views/content/index.html.erb

```diff
@@ -84,18 +86,20 @@
     </div>
     <div class="govuk-grid-column-three-quarters">
       <div class="table-wrapper">
+        <%=  %>
         <%= GovukPublishingComponents::AppHelpers::TableHelper.helper(
             self,
             '<h1 class="table-header">Showing <span class="table-header__param">21</span> to <span class="table-header__param">40</span> of <span class="table-header__param">587</span> results from <span class="table-header__param">UK Visas and Immigration</span> in <span class="table-header__param">guidance</span>.</h1>'.html_safe,
-            sortable: true
+            sortable: false
           ) do |t| %>
             <%= t.head do %>
               <!-- TODO: Once sorting is supported, fill in the href and sort_direction header arguments with proper values -->
-              <%= t.header("Page title", href: '#', sort_direction: nil) %>
-              <%= t.header("Content type", href: '#', sort_direction: nil) %>
-              <%= t.header("Unique pageviews", href: '#', sort_direction: 'descending') %>
-              <%= t.header("User satisfaction score", href: '#', sort_direction: nil) %>
-              <%= t.header("Searches from page", href: '#', sort_direction: nil) %>
+              <%# t.header("Page title", href: '#', sort_direction: nil) %>
+              <%= t.header("Page title") %>
+              <%= t.header("Content type") %>
+              <%= t.header("Unique pageviews<br>" "#{image_tag('question_mark_default.svg', alt: 'Help', title: "#{t "metrics.upviews.summary"}")}".html_safe) %>
+              <%= t.header("User satisfaction score<br>" "#{image_tag('question_mark_default.svg', alt: 'Help', title: "#{t "metrics.satisfaction.summary"}")}".html_safe) %>
+              <%= t.header("Searches from page<br>" "#{image_tag('question_mark_default.svg', alt: 'Help', title: "#{t "metrics.searches.summary"}")}".html_safe) %>
```

Comment:
Are the double quotes here `<br>" "#{` intentional? I'm guessing that it would work the same without them.

Reply:
They were, but you're right, they weren't needed.